src/build.sh

```diff
@@ -151,6 +162,20 @@ function parse_build_options()
         options_values['MENU_CONFIG']="${build_config[menu_config]:-nconfig}"
         shift
         ;;
+      --cpu-util | --cpu-scale | -c)
+        [[ ! "$2" =~ [0-9]+ ]] && {
+          options_values['ERROR']="$2"
+          return 22
+        }
+        [ "$2" -gt 100 ] &&
```

Comment:
Same as the previous comment, and why not use `[[ .. ]]`?

Reply:
I feel that using bash syntax for something as simple is kinda unnecessary (i.e. `[[` is a bashism, and not a simple alternative to `[`)